docs/docusaurus/docs/reference/expectations/result_format.md

```diff
@@ -19,6 +19,8 @@ When using a dictionary, `result_format` can include the following keys:
 
 - `partial_unexpected_count`: Sets the number of results to include in `partial_unexpected_count``, if applicable. Set the value to zero to suppress the unexpected counts.
 
+- `unexpected_metrics_with_values`: When running validations, a set of unexpected results' indices and values is returned.  Setting this value to `False` suppresses values from the output to only have indices (default is `True`).
```

Comment:
Hi @MMoussabbih! Would you consider a change to the naming of the parameter to help clarify the functionality? Note that the new name also flips the default value.
```suggestion
- `exclude_unexpected_values`: When running validations, a set of unexpected results' indices and values is returned.  Setting this value to `True` suppresses values from the output to only have indices (default is `False`).
```

Reply:
Hello @anthonyburdi,
Thanks for your comment. I have updated the parameter name as you mentioned in my last commit 👍